### PR TITLE
Set Default Content moderation_state to published on import

### LINKED ISSUE #248: Access denied on home page after enabling govcms8_default_content
**Steps to reproduce**

1. Install fresh (current version at time of writing is 1.0.0-beta2)
2. Enable the "GovCMS 8 Default Content" module
3. Visit the site as anonymous -> Access denied 👎 
4. Grant "View any unpublished content" permission to anonymous users
5. Visit the site as anonymous -> works 👍 
6. Revoke "View any unpublished content" permission from anonymous users
7. Edit the "GovCMS8 Starter" page -> notice how it's set to draft
8. Set to "Needs review", then set to "Published"
9. Visit the site as anonymous -> works 👍 

Furthermore, all the pages linked from the footer menu are also set to draft, which causes the entire footer to be missing when viewing the home page as anonymous.

Same goes for all the nodes listed in the "Content example (demo)" section.

**Expected behavior**

When a person is demoing a fresh installation of GovCMS8 after having enabled the govcms8_default_content module, they should not get "access denied" on the front page, and all demo components should be accessible by anonymous users.

The "GovCMS8 Starter" page and all the demo content listed or linked to from the home page should be published OOTB when the "GovCMS 8 Default Content" module is enabled, so that all the features of the demo home page can be viewed by anonymous users.

List of nodes that need to be set to published OOTB:

- /homepage
- /about-us
- /accessibility
- /copyright
- /disclaimers
- /privacy-statement
- /blog/example
- /event/example
- /news-and-media/example
- /freedom-of-information/example
- /standard-page-example

### PR DESCRIPTION
**Pull request by tobybellwood**
govcms8_default_content import was implemented before content_moderation was included, so had no default setting for imported content. This PR sets it all to "published" to make it all viewable.

This fixes #248 

Ref:GOVCMSD8-298

### CODE REVIEW

File: modules/custom/optional/govcms8_default_content/src/InstallHelper.php [hunk 1]
```diff
@@ -113,6 +113,7 @@ protected function importPages() {
             $values['body'] = [
               'value' => $body,
               'format' => 'rich_text',
+              'moderation_state' => 'published', 
             ];
```
Comment (gargsuchi) [inline modules/custom/optional/govcms8_default_content/src/InstallHelper.php]: @tobybellwood This line is setting the moderation state of the body field only.

**Comment by gargsuchi** [comment]
Updated the PR.

**Review by govhosting — APPROVED** [review]

**Comment by gargsuchi** [comment]
@tobybellwood  should we close this PR - considering that the code is now merged into the 1.x branch? 
https://github.com/govCMS/govCMS8/blob/1.x/modules/custom/optional/govcms8_default_content/src/InstallHelper.php

**Comment by tobybellwood** [comment]
closing - already merged!

**Pull request closed without merging**